checkov/terraform/checks/resource/azure/ACRAdminAccountDisabled.py

```diff
@@ -0,0 +1,21 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ACRAdminAccountDisabled(BaseResourceCheck):
```

Comment:
I recommend to use the `BaseResourceNegativeValueCheck` as base class, then `checkov` can leverage additional logic to make the `check` more resilient, here is an example https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/MemoryDBClusterIntransitEncryption.py

Reply:
Thanks for the recommendation and example, I can make another commit if that's the best way forward.